modules/manage/partials/authentication.adoc

```diff
@@ -687,6 +690,47 @@ pandaproxy_client:
 ----
 endif::[]
 
+[[plain]]
+==== SASL/PLAIN
+
+You can configure Kafka clients to authenticate using either SASL/SCRAM or SASL/PLAIN with a
+single account using the same username and password. Unlike SASL/SCRAM, which uses a challenge response with hashed
+credentials, SASL/PLAIN transmits plaintext passwords, but requires TLS for external encryption.
+
+If you have existing PLAIN Kafka clients and applications, you can migrate to Redpanda without updating
+your application by creating local Redpanda SCRAM accounts and enabling PLAIN as an authentication mechanism.
+
+NOTE: Clusters configured with _only_ a SASL/PLAIN mechanism are not supported.
```

Comment:
We should probably note that it's not a recommended. We log this: "SASL/PLAIN is enabled. This is insecure and not recommended for production".